lib/solidus_subscriptions/configuration.rb

```diff
@@ -4,14 +4,14 @@ module SolidusSubscriptions
   class Configuration
     attr_accessor(
       :maximum_total_skips, :maximum_reprocessing_attempts, :churn_buster_account_id,
-      :churn_buster_api_key,
+      :churn_buster_api_key, :disable_admin_menu_link
     )
 
     attr_writer(
       :success_dispatcher_class, :failure_dispatcher_class, :payment_failed_dispatcher_class,
       :out_of_stock_dispatcher, :maximum_successive_skips, :reprocessing_interval,
       :minimum_cancellation_notice, :processing_queue, :subscription_line_item_attributes,
-      :subscription_attributes, :subscribable_class,
+      :subscription_attributes, :subscribable_class
```

Comment:
Usually it's good to have trailing commas in order to minimise line changes when adding new elements 😉

Reply:
Good call. Thanks, @nirebu :D